test/test_transformers.py

```diff
@@ -73,6 +82,15 @@ def get_rtol(true_value: torch.Tensor, computed_value: torch.Tensor) -> float:
         "enable_math": False, "enable_flash": False, "enable_mem_efficient": True}
 }
 
+def get_platform_specific_spda():
+    if TEST_CUDA and SM80OrLater:
+        return [SDPBackend.FLASH_ATTENTION, SDPBackend.EFFICIENT_ATTENTION]
+    if TEST_WITH_ROCM:
+        return [SDPBackend.FLASH_ATTENTION]
+    if TEST_CUDA:
+        return [SDPBackend.EFFICIENT_ATTENTION]
```

Comment:
Could be rewritten slightly better to have `TEST_WITH_ROCM` condition first and combine the TEST_CUDA conditions?